data/projects.json

```diff
@@ -30,13 +30,13 @@
         "tags": "html, css, jsp, servlets, mysql"
     },
     {
-        "name": "currency-converter",
+        "name": "Roshaen",
         "display": "Currency Convertor",
         "description": "A simple website using HTML, CSS, Javascript that converts one currency amount to other",
         "author": "Roshaen",
         "category": "website",
         "level": "basic",
-        "site": "https://roshaen.github.io/currency",
+        "site": "https://github.com/Roshaen",
         "tags": "html, css, js"
     },
```

Comment:
Could you help me to remove this data?

Reply:
Hi @ming-tsai
I've removed lines 32-41 on [this](https://github.com/MakeContributions/ideahub/pull/100/commits/99614bb4bd3fc1a98f40ab3b8a4a00860453b460) commit. Is it okay?